packages/nimble-components/src/utilities/unit-format/default-unit-format.ts

```diff
@@ -69,20 +61,19 @@ export class DefaultUnitFormat extends UnitFormat {
                 unit.scaleFactor,
                 unit.scaledUnitFormatFactory({
                     locale,
-                    intlNumberFormatOptions: this.defaultIntlNumberFormatOptions
-                })
-            );
-            this.leadingZeroScaledUnitFormatters.set(
-                unit.scaleFactor,
-                unit.scaledUnitFormatFactory({
-                    locale,
-                    intlNumberFormatOptions:
-                        this.leadingZeroIntlNumberFormatOptions
+                    // Hack to avoid ts error about roundingPriority not being a known option.
+                    // It has been supported by browsers since 8/23, but TypeScript still hasn't
+                    // added it to the type definitions. See https://github.com/microsoft/TypeScript/issues/56269
+                    intlNumberFormatOptions: {
+                        ...this.defaultIntlNumberFormatOptions,
+                        roundingPriority: 'lessPrecision'
+                    } as Intl.NumberFormatOptions
                 })
             );
         }
         this.exponentialScaledUnitFormatter = unitScale.baseScaledUnit.scaledUnitFormatFactory({
             locale,
+            // Same as above comment
```

Comment:
It isn't clear to me what comment this is referring to (and how that applies to this line). Can you clarify this comment?

Reply:
Whoops, should have removed that.